mail-api.js

```diff
@@ -96,6 +96,34 @@ module.exports = {
     });
   },
   
+
+  getMessageAttachmentId: function(parameters, callback) {
```

Comment:
This function doesn't return the attachment ID, it returns a collection of attachments. A better name for this function would be `getMessageAttachments`.